boost-common/src/main/java/io/openliberty/boost/common/utils/SpringBootUtil.java

```diff
@@ -111,6 +111,44 @@ public static File copySpringBootUberJar(File artifact, BoostLoggerI logger) thr
         return null;
     }
 
+    /**
```

Comment:
I think the method comments for this method and the previous one need to contain more details. I'm wondering why we have two methods to copy the Spring Boot Uber JAR. When would I use this one vs the previous one?

Reply:
I guess the `copySpringBootUberJarFile` method is really just a helper for `copySpringBootUberJar`. The first method will conditionally perform the actual copy, while the second does the copy back. If you recall, the logic was written this way to allow re-execution of the `package` goal with a Liberty Spring Boot artifact already generated. This means that we need the same "entry point" for both behaviors since we are unsure of the state of the build previous build (if any) at this point in the execution. 

It looks like we don't call `copySpringBootUberJarFile` anywhere outside of this method, so I think it's sufficient to change the visibility to `private` and indicate in the comments that this is a helper for the publicly visible `copySpringBootUberJar`.